hw/ip/edn/data/edn.hjson

```diff
@@ -243,6 +249,22 @@
         }
       ]
     },
+    {
+      name: "RECOV_ALERT_STS",
+      desc: "Recoverable alert status register",
+      swaccess: "rw0c",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "12",
```

Comment:
i have to ask...why start at bit 12?

Reply:
yeah looks odd, but in trying to be somewhat consistent between the 3 entropy complex blocks, reserving 0-11 bits for enable field detected alerts. Starting at 12 for additional alert sources, such as health checks in ES, and entropy bus comparison fails, etc.